src/app/components/transaction/components/transaction-title.tsx

```diff
@@ -0,0 +1,35 @@
+import { useEffect, useRef, useState } from 'react';
+
+import { Tooltip } from '@app/components/tooltip';
+import { Title } from '@app/components/typography';
+
+interface TransactionTitleProps {
+  title: string;
+}
+export function TransactionTitle(props: TransactionTitleProps) {
+  const { title } = props;
+  const ref = useRef<HTMLDivElement | null>(null);
+  const [isEllipsisActive, setIsEllipsisActive] = useState(false);
+
+  useEffect(() => {
+    const element = ref.current!;
+    setIsEllipsisActive(element?.scrollWidth > element?.clientWidth);
+    const onResize = () => setIsEllipsisActive(element?.scrollWidth > element?.clientWidth);
+    window.addEventListener('resize', onResize);
+    return () => window.removeEventListener('resize', onResize);
+  }, []);
```

Comment:
This seems like something that should be turned into its own `useOnResizeListener` hook.

Also, are we _sure_ there's no way to do this without listening for resize events in js? There's no CSS-only solution? cc/ @andresgalante

Reply:
I did do quite a bit of searching around on this and followed suggestions, would be interested to know if there is another way?